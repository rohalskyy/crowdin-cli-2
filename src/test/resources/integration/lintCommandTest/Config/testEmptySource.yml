"project_id" : "1"
"api_token" : "a"
"base_path" : ""
"base_url": ""

"preserve_hierarchy": true

files: [
    {
        "translation" : "%two_letters_code%.md",
    }
]
