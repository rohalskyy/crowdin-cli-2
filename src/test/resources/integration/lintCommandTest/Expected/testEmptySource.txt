Crowdin CLI initialization has failed. Please check your configuration file and credentials.
Configuration file is invalid. Check the following parameters in your configuration file:
	- `Source` parameter is empty